code/src/Components/ThoughtElements.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import Moment from 'moment';
+
+
+
+const ThoughtElement = ({ thought, handleLike }) => {
+  console.log(thought)
```

Comment:
There's only one console log here but all 20 thoughts are logged when there is a new post so it quite quickly becomes a very full console.